core/controllers/creator_dashboard.py

```diff
@@ -264,9 +250,13 @@ def _round_average_ratings(rating):
 
         last_week_stats = (
             user_services.get_last_week_dashboard_stats(self.user_id))
-        if last_week_stats and last_week_stats.get('average_ratings'):
-            last_week_stats['average_ratings'] = (
-                _round_average_ratings(last_week_stats['average_ratings']))
+        if last_week_stats:
+            datetime_of_stats = list(last_week_stats.keys())[0]
```

Comment:
The first two parts are done.

Regarding the third part, len(last_week_stats.keys()) will always be one. You can see the logic [here](https://github.com/oppia/oppia/blob/develop/core/domain/user_services.py#L1561). Hence there can never be a condition when the length is not 1(except None of course!). This will in turn create a statement that can not be tested which we do not want

Reply:
Hmm. In that case, you could use the assertion approach that I suggested in another PR. We need to ensure that, if the code changes under us somehow, we catch the error.

But I'm not sure I buy your argument that this will create untestable code. You could mock user_services.get_last_week_dashboard_stats() to give you an invalid dictionary.

Reply:
Done